packages/libs/error-stack/.gitignore

```diff
@@ -1,2 +1,3 @@
 target/
 Cargo.lock
+.idea
```

Comment:
```suggestion

```

Reply:
I assume you added this because you opened the error-stack folder as a root, rather than the repo? `.idea` is [already ignored in the root](https://github.com/hashintel/hash/blob/48f3b82744a91e7e934cbfc356eb1e7017b35e10/.gitignore#L79) `.gitignore`

If you're opening the error-stack folder as root, then us suggesting you remove this line might be frustrating for local development. I recommend you consider some of the approaches [outlined here](https://blog.martinhujer.cz/dont-put-idea-vscode-directories-to-projects-gitignore/).